nemo/collections/nlp/models/machine_translation/mt_enc_dec_model.py

```diff
@@ -325,7 +331,7 @@ def _setup_dataloader_from_config(self, cfg: DictConfig):
         )
 
     @torch.no_grad()
-    def translate(self, text: List[str], source_lang: str = None, target_lang: str = None) -> List[str]:
```

Comment:
It looks like we have to keep these for backwards compatiblity. Can we check if they are None? And if so then we use the model attributes?